### ISSUE TITLE
Incorrect focus on telemetry screen modal dialog

### ISSUE THREAD

**Issue by joestegman425** [microsoft/accessibility-insights-windows#1897]
**Describe the bug**
When the A14Win telemetry modal is displayed, focus is on the "OK" button which leaves screen reader users confused. The focus should be on the "enable telemetry checkbox".

**To Reproduce**
Install a fresh copy of AI4Win and run. The telemetry screen will display. Notice focus is on the OK button.

**Expected behavior**
Focus should be on the checkbox.
